src/index.js

```diff
@@ -297,6 +300,7 @@ class Dropzone extends React.Component {
         onDragOver={this.onDragOver}
         onDragLeave={this.onDragLeave}
         onDrop={this.onDrop}
+        ref={node => (this.node = node)}
```

Comment:
This will generate a new function each time. Instead, define a function outside of the scope and pass a reference to it.